simpleclient_spring_boot/src/main/java/io/prometheus/client/spring/boot/PrometheusMvcEndpoint.java

```diff
@@ -24,7 +24,7 @@ public PrometheusMvcEndpoint(PrometheusEndpoint delegate) {
 
   @RequestMapping(
           method = {RequestMethod.GET},
-          produces = {TextFormat.CONTENT_TYPE_004}
+          produces = {TextFormat.CONTENT_TYPE_004, TextFormat.CONTENT_TYPE}
```

Comment:
I think what's needed here is a wildcard, as this only solves it for one content type if I understand this correctly